ui/package.json

```diff
@@ -5,7 +5,12 @@
   "scripts": {
     "ng": "nx",
     "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main",
+    "generate:vars": "node node_modules/ng-node-environment/index.js --out='./apps/ui/src/environments/environment.generated.ts' && eslint --fix ./apps/ui/src/environments/environment.generated.ts",
+    "make:requirements": "npm run generate:vars",
+    "prestart": "npm run make:requirements",
     "start": "nx serve",
```

Comment:
Please update the `start` section with the required commands to run the app without issues (like generating vars, making requirements, and so on). It'll be much easier to develop new things running only one command.